docs/managing-cpp-deps.md

```diff
@@ -0,0 +1,48 @@
+---
+id: managing-cpp-deps
+title: Managing C++ dependencies
+---
+
+Details to consider when consuming a Visual C++ project as a source dependency of your app.
```

Comment:
A major part of the RNW demographic will be familiar with Visual Studio project structure. From what I've observed, others have expertise in JS, and rely on tools like "autolinking" to manage dependencies for them.

That audience may not know, that under the hood, their npm package dependency brings in a vcxproj. The term the community would recognize for these npm dependencies bringing in native code is "community modules". Since that has the potential to be such a common case, could we call them out by name throughout the document?

E.g.

```suggestion
Details to consider when consuming community modules or other Visual C++ projects.
```